### PR TITLE
Add basic test scripts for hana and grunt

### PR DESCRIPTION
**Pull request by eballetbo**

### CODE REVIEW

File: boards/google,hana [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+assert_driver_present cros-ec-debugfs-driver-present cros-ec-debugfs
+assert_device_present cros-ec-debugfs-probed cros-ec-debugfs cros-ec-debugfs.*
+assert_sysfs_attr_present cros-ec-debugfs-attr-console_log /sys/kernel/debug/cros_ec/console_log
+assert_sysfs_attr_present cros-ec-debugfs-attr-pdinfo /sys/kernel/debug/cros_ec/pdinfo
+
+assert_driver_present cros-ec-dev-driver-present cros-ec-dev
+assert_device_present cros-ec-dev-probed cros-ec-dev cros-ec-dev.*
+
+assert_driver_present cros-ec-lightbar-driver-present cros-ec-lightbar
+# assert_device_present cros-ec-lightbar-driver-present is expected to fail.
```
Comment (gctucker) [inline boards/google,hana]: Will this get fixed with a kernel patch?

Generally speaking, it's OK to have tests that fail initially and then start passing.  The tests that have always failed are not tracked as regressions and appear as "unknown".  So even if this gets fixed in next and mainline but fails on stable branches it's fine from a KernelCI perspective.

**Review by gctucker — COMMENTED** [review]
LGTM - will get this built and tested on staging.

Depends on #5.

File: boards/google,hana (hunk 1 above)
Comment (eballetbo) [reply boards/google,hana]: This test will never pass on this device so I removed it.

**Comment by gctucker** [comment]
Tested with a manually modified ramdisk.

On `hana`, before and after:
https://lava.collabora.co.uk/results/2652941/1_bootrr
https://lava.collabora.co.uk/results/2655316/1_bootrr

On `grunt`, before and after, in combination with #5:
https://lava.collabora.co.uk/results/2644447/1_bootrr
https://lava.collabora.co.uk/results/2655312/1_bootrr

**Review by gctucker — APPROVED** [review]

**Pull request merged**